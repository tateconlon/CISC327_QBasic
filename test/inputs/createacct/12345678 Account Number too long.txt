login
agent
createacct
12345678
TestAccount
logout