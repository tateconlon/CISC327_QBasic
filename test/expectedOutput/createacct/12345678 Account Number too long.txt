login as agent successful
Account number 12345678 not valid
Transaction summary written and system logged out successfully